src/types/index.ts

```diff
@@ -6,6 +6,7 @@ import type { AllowedHTTPMethods, BasicAuth, CorsOptions, RateLimiter, RequestSi
 
 export type Ssg = {
   hashScripts?: boolean;
+  hashStyles?: boolean;
```

Comment:
suggestion: Could you also please add it to the configuration documentation page where the default config is displayed?